### PR TITLE
finish collections

### PR DESCRIPTION
**Pull request by kipijz**

### CODE REVIEW

File: src/main/java/io/codelex/collections/practice/FlightPlanner.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Scanner;
+
+import static java.nio.file.Files.readAllLines;
+
+public class FlightPlanner {
+    private static final Charset charset = Charset.defaultCharset();
```
Comment (NilsKula) [inline src/main/java/io/codelex/collections/practice/FlightPlanner.java:16]: static final in caps

File: src/main/java/io/codelex/collections/practice/Histogram.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    }
+
+    private static void setHistogramStars() throws URISyntaxException, IOException {
+        final String scores = fileContent();
+        String[] scoresArray = scores.split(" ");
+        for (String s : scoresArray) {
+            if (Integer.parseInt(s) <= 9 && Integer.parseInt(s) >= 0) {
+                zeroToNine.append("*");
+            }
+            if (Integer.parseInt(s) <= 19 && Integer.parseInt(s) >= 10) {
```
Comment (NilsKula) [inline src/main/java/io/codelex/collections/practice/Histogram.java:67]: pietiek ar `Integer.parseInt(s) <= 19`, zini jau ka nav zem 10

File: src/main/java/io/codelex/collections/practice/phonebook/PhoneEntry.java [hunk 3]
```diff
@@ -0,0 +1,6 @@
+package io.codelex.collections.practice.phonebook;
+
+public class PhoneEntry {
+    String name;
```
Comment (NilsKula) [inline src/main/java/io/codelex/collections/practice/phonebook/PhoneEntry.java:4]: private

**Review by NilsKula — CHANGES_REQUESTED** [review]
👍

**Comment by kipijz** [comment]
paldies, izlabots

**Review by NilsKula — APPROVED** [review]

**Review by NilsKula — APPROVED** [review]
merge it :+1:

**Review by janistumpelis — APPROVED** [review]

**Pull request merged**